python/sdk/merlin/requirements.py

```diff
@@ -0,0 +1,285 @@
+import os
```

Comment:
This file is a simplified version of https://github.com/mlflow/mlflow/blob/master/mlflow/utils/environment.py